examples/psp22/lib.rs

```diff
@@ -48,9 +100,66 @@ pub mod my_psp22 {
             }
             Ok(())
         }
+
+        fn _after_token_transfer(
+            &mut self,
+            from: Option<&AccountId>,
+            to: Option<&AccountId>,
+            amount: &Balance,
+        ) -> Result<(), PSP22Error> {
+            InternalImpl::_after_token_transfer(self, from, to, amount)
+        }
     }
 
-    impl PSP22 for Contract {}
+    impl PSP22Impl for Contract {}
+
+    impl PSP22 for Contract {
+        #[ink(message)]
```

Comment:
do we need to have #[ink(message)] here?

Reply:
yes it is necessary